ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-authorization.j2

```diff
@@ -1,22 +1,37 @@
-{% if aaa_authorization is defined and aaa_authorization is not none %}
+{% if aaa_authorization is arista.avd.defined %}
 
 ## AAA Authorization
 
 ### AAA Authorization Summary
 
 | Type | User Stores |
 | ---- | ----------- |
-{%     if aaa_authorization.exec.default is defined %}
+{%     if aaa_authorization.exec.default is arista.avd.defined %}
 | Exec | {{ aaa_authorization.exec.default }} |
 {%     endif %}
 
-{%     if aaa_authorization.config_commands is defined and aaa_authorization.config_commands == true %}
+{%     if aaa_authorization.config_commands is arista.avd.defined(true) %}
 Authorization for configuration commands is enabled.
 {%     else %}
 Authorization for configuration commands is disabled.
 {%     endif %}
 
-{%     if aaa_authorization.serial_console is defined and aaa_authorization.serial_console == true %}
+### AAA Authorization Privilege Levels Summary
+
+{%     if aaa_authorization.privilege is arista.avd.defined or aaa_authorization.commands.all_default is arista.avd.defined %}
```

Comment:
Please move this ```if test ``` above line 19. Because we don't want to generate even the title if a test fails. 

You can do something like:
line18: {% if aaa_authorization...... %}
line19: <blank line>
line20: ### AAA Authorization Privilege Levels Summary
line21: <blank line>
..... rest of the code .........

We usually follow this format: ```if test``` then blank line, ```title```, blank line, then code.

Reply:
Also this should be ```aaa_authorization.commands.privilege```. You had a typo ```aaa_authorization.privilege```